docs(guides/installation): introduce sections for nightlies and dev branch (#93)
embark-framework/embark-site: source/docs/installation.md

<pre><code class="shell">$ npm -g install embark</code></pre>

After that, `embark` should be available as a global command in our terminal of choice. Let's verify this by running the following command:

<pre><code class="shell">$ embark --version</code></pre>

At the time of writing this guide, the output looked like this:

<pre><code class="shell">3.2.1</code></pre>

## Installing Embark @next

While running `npm install -g embark` will give us the latest and great stable version of Embark, we're sometimes interested in trying out features that are in active development. The Embark team maintains a `next` distribution tag on npm that can be used to install versions of Embark that aren't stable.

<pre><code class="shell">$ npm -g install embark@next</code></pre>

## Installing Embark's latest GitHub version

> If we're interested in getting whatever has landed last in the code base, we can go ahead and install the latest version of Embark's GitHub repository like this:

Reviewer: ```suggestion
If we're interested in getting whatever has landed last in the code base, we can install directly from Embark's GitHub repository like this:
```

<pre><code class="shell">$ npm -g install embark-framework/embark</code></pre>

{% note warn Warning: %}

> Installations with `@next` or directly from the `develop` branch on GitHub areconsidered unstable and may have bugs, so please proceed with caution.

Reviewer 2: ```suggestion
Installations with `@next` or directly from the `develop` branch on GitHub are considered unstable and may have bugs, so please proceed with caution.
```

Author: Eagle eye!

<pre><code class="shell">3.2.1</code></pre>

## Installing Embark @next

> While running `npm install -g embark` will give us the latest and great stable version of Embark, we're sometimes interested in trying out features that are in active development. The Embark maintains a `next` distribution tag on npm that can be used to install versions of Embark that aren't stable.

Reviewer 2: "The Embark", did you mean "The Embark team"?

Author: Yes good catch!

<pre><code class="shell">$ npm -g install embark@next</code></pre>

## Installing Embark's latest nightly

If you're interested in getting whatever has landed last in the code base, we can go ahead and install the latest version of Embark's GitHub repository like this:

> <pre><code class="shell">$ npm -g install github.com/embark-framework/embark</code></pre>

Reviewer: Re: the *latest nightly* section:

I don't think it's correct to refer to it as a *nightly*, since the develop branch could change at any time, multiple times, on any given day, while it may not change at all on the weekends.

Maybe change the section header to *Installing from GitHub*.

So how about:

```
If you're interested in trying out the latest changes to Embark, you can install directly
from the `develop` branch (the default branch) on GitHub:

<pre><code class="shell">$ npm -g install embark-framework/embark</code></pre>
```

{% note warn Warning: %}

> Keep in mind that both, latest nightlies as well as the current `@next` are considered unstable and may have some bugs. Please install those at your own risk.

Reviewer: How about?

```
Installations with `@next` or directly from the `develop` branch on GitHub are
considered unstable and may have bugs, so please proceed with caution.
```

Author: This is really good!

{% endnote %}

> Awesome! We're all set up. Let's build our first decentralized application!

Reviewer: General thought/observation:

the style might be better overall if you / you're *-OR-* we / we're are used consistently, vs. switching back and forth between them.
